File: Samples-Xamarin/Adapters/ChartExamples.cs
using System;
using Java.Util;

namespace Samples
{
	public class ChartExamples : Java.Lang.Object, ExamplesProvider {

		private LinkedHashMap chartExamples;
        public static double[] randomDoubleValues = new double[] { 52.00d, 60.00d, 77.00d, 50.00d, 56.00d, 75.00d, 85.00d, 15.00d, 25.00d, 5.00d, 10.00d, 20.00d, 30.00d, 40.00d, 45.00d, 60.00d, 80.00d, 50.00d, 35.00d, 70.00d, 77.00d};
        public static int[] randomIntValues = new int[] { 70, 60, 30, 40, 50, 20, 5, 10, 15, 20, 25, 35, 55, 65, 75, 85, 80, 10, 15, 20, 30, 45, 60, 80, 50, 55, 70, 60, 30, 40, 50, 20, 5, 10, 15, 20, 25, 35, 55, 65, 75, 85, 80, 10, 15, 20, 30, 45, 60, 80, 50, 55, 70, 60, 30, 40, 50, 20, 5, 10, 15, 20, 25, 35, 55, 65, 75, 85, 80, 10, 15, 20, 30, 45, 60, 80, 50, 55, 70, 60, 30, 40, 50, 20, 5, 10, 15, 20, 25, 35, 55, 65, 75, 85, 80, 10, 15, 20, 30, 45, 60, 80, 50, 55 };

        public ChartExamples(){

			this.chartExamples = this.getChartExamples();
		}


		public String ControlName() {
			return "Chart";
		}


		public LinkedHashMap Examples(){
			return this.chartExamples;
		}

		private LinkedHashMap getChartExamples(){
			LinkedHashMap chartExamples = new LinkedHashMap();

			ArrayList result = new ArrayList();

			result.Add(new AreaSeriesFragment());
			result.Add(new LineSeriesFragment());
			result.Add(new CandleStickSeriesFragment());
			result.Add(new DoughnutSeriesFragment());
			result.Add(new HorizontalBarSeriesFragment());
			result.Add(new IndicatorSeriesFragment());
			result.Add(new OhlcSeriesFragment());
			result.Add(new PieSeriesFragment());
			result.Add(new ScatterBubbleSeriesFragment());
			result.Add(new ScatterPointSeriesFragment());
			result.Add(new SplineAreaSeriesFragment());
			result.Add(new SplineSeriesFragment());
			result.Add(new StackAreaSeriesFragment());
			result.Add(new StackBarSeriesFragment());
			result.Add(new StackSplineAreaSeriesFragment());
			result.Add(new VerticalBarSeriesFragment());

			chartExamples.Put("Series", result);

			result = new ArrayList();

			result.Add(new ChartLegendFragment());
			result.Add(new GridFeatureFragment());
			result.Add(new PalettesFragment());

			chartExamples.Put("Features", result);

			result = new ArrayList();

			result.Add(new PanAndZoomFragment());
			result.Add(new SelectionBehaviorFragment());
			result.Add(new TooltipBehaviorFragment());
			result.Add(new TrackballBehaviorFragment());

			chartExamples.Put("Behaviors", result);

			result = new ArrayList();

			result.Add(new DateTimeContinuousAxisFragment());
			result.Add(new MultipleAxesFragment());

			chartExamples.Put("Axes", result);

			result = new ArrayList();

			result.Add(new GridLineAnnotationFragment());
			result.Add(new PlotBandAnnotationFragment());

			chartExamples.Put("Annotations", result);

			return chartExamples;
		}
	}
}

File: Samples-Java/Samples/src/main/java/activities/ChartExamples.java
package activities;

import java.util.ArrayList;
import java.util.LinkedHashMap;

import fragments.chart.annotations.GridLineAnnotationFragment;
import fragments.chart.annotations.PlotBandAnnotationFragment;
import fragments.chart.axes.DateTimeContinuousAxisFragment;
import fragments.chart.axes.MultipleAxesFragment;
import fragments.chart.behaviors.PanAndZoomFragment;
import fragments.chart.behaviors.SelectionBehaviorFragment;
import fragments.chart.behaviors.TooltipBehaviorFragment;
import fragments.chart.behaviors.TrackBallBehaviorFragment;
import fragments.chart.features.ChartLegendFragment;
import fragments.chart.features.GridFeatureFragment;
import fragments.chart.features.PalettesFragment;
import fragments.chart.series.AreaSeriesFragment;
import fragments.chart.series.CandleStickSeriesFragment;
import fragments.chart.series.DoughnutSeriesFragment;
import fragments.chart.series.HorizontalBarSeriesFragment;
import fragments.chart.series.IndicatorSeriesFragment;
import fragments.chart.series.LineSeriesFragment;
import fragments.chart.series.OhlcSeriesFragment;
import fragments.chart.series.PieSeriesFragment;
import fragments.chart.series.ScatterBubbleSeriesFragment;
import fragments.chart.series.ScatterPointSeriesFragment;
import fragments.chart.series.SplineAreaSeriesFragment;
import fragments.chart.series.SplineSeriesFragment;
import fragments.chart.series.StackAreaSeriesFragment;
import fragments.chart.series.StackBarSeriesFragment;
import fragments.chart.series.StackSplineAreaSeriesFragment;
import fragments.chart.series.VerticalBarSeriesFragment;

public class ChartExamples implements ExamplesProvider {

    private LinkedHashMap<String, ArrayList<ExampleFragment>> chartExamples;

    public ChartExamples(){
        this.chartExamples = this.getChartExamples();
    }

    @Override
    public String controlName() {
        return "Chart";
    }

    @Override
    public LinkedHashMap<String, ArrayList<ExampleFragment>> examples(){
        return this.chartExamples;
    }

    private LinkedHashMap<String, ArrayList<ExampleFragment>> getChartExamples(){
        LinkedHashMap<String, ArrayList<ExampleFragment>> chartExamples = new LinkedHashMap<String, ArrayList<ExampleFragment>>();

        ArrayList<ExampleFragment> result = new ArrayList<ExampleFragment>();

        result.add(new AreaSeriesFragment());
        result.add(new LineSeriesFragment());
        result.add(new CandleStickSeriesFragment());
        result.add(new DoughnutSeriesFragment());
        result.add(new HorizontalBarSeriesFragment());
        result.add(new IndicatorSeriesFragment());
        result.add(new OhlcSeriesFragment());
        result.add(new PieSeriesFragment());
        result.add(new ScatterBubbleSeriesFragment());
        result.add(new ScatterPointSeriesFragment());
        result.add(new SplineAreaSeriesFragment());
        result.add(new SplineSeriesFragment());
        result.add(new StackAreaSeriesFragment());
        result.add(new StackBarSeriesFragment());
        result.add(new StackSplineAreaSeriesFragment());
        result.add(new VerticalBarSeriesFragment());

        chartExamples.put("Series", result);

        result = new ArrayList<ExampleFragment>();

        result.add(new ChartLegendFragment());
        result.add(new GridFeatureFragment());
        result.add(new PalettesFragment());

        chartExamples.put("Features", result);

        result = new ArrayList<ExampleFragment>();

        result.add(new PanAndZoomFragment());
        result.add(new SelectionBehaviorFragment());
        result.add(new TooltipBehaviorFragment());
        result.add(new TrackBallBehaviorFragment());

        chartExamples.put("Behaviors", result);

        result = new ArrayList<ExampleFragment>();

        result.add(new DateTimeContinuousAxisFragment());
        result.add(new MultipleAxesFragment());

        chartExamples.put("Axes", result);

        result = new ArrayList<ExampleFragment>();

        result.add(new GridLineAnnotationFragment());
        result.add(new PlotBandAnnotationFragment());

        chartExamples.put("Annotations", result);

        return chartExamples;
    }
}
